helm-charts/charts/kube-starrocks/charts/starrocks/values.yaml

```diff
@@ -408,3 +408,54 @@ configMaps: []
   #     key: |
   #       this is the content of the configmap
   #       when mounted, key will be the name of the file
+
+# specify the fe proxy deploy or not.
+feProxy:
+  # specify the fe proxy deploy or not.
+  enabled: true
+  replicas: 1
+  image:
+    repository: nginx
+    tag: 1.24.0
+  resources:
+    limits:
+      cpu: 1
+      memory: 2Gi
+    requests:
+      cpu: 1
+      memory: 2Gi
+  # set the resolver for nginx server
+  resolver: kube-dns.kube-system.svc.cluster.local
+  service:
+    # the fe proxy service type, only supported ClusterIP, NodePort, LoadBalancer
+    type: LoadBalancer
+    # the loadBalancerIP for static ip config when the type=LoadBalancer and loadbalancerIp is not empty.
+    loadbalancerIP: ""
+  # If specified, the pod's nodeSelector，displayName="Map of nodeSelectors to match when scheduling pods on nodes"
+  # Ref: https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector
+  nodeSelector: {}
+    # kubernetes.io/arch: amd64
+    # kubernetes.io/os: linux
+  # affinity for fe proxy pod scheduling.
+  affinity: {}
+    # nodeAffinity:
+    #   requiredDuringSchedulingIgnoredDuringExecution:
+    #     nodeSelectorTerms:
+    #     - matchFields:
+    #       - key: metadata.name
+    #         operator: In
+    #         values:
+    #         - target-host-name
+  # Node tolerations for fe proxy pod scheduling to nodes with taints
+  # Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/
+  tolerations: []
+    # - key: "key"
+    #   operator: "Equal|Exists"
+    #   value: "value"
+    #   effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
+  # This ConfigMap store the nginx.conf for fe proxy.
+  # If configMapName is empty, operator will use its default nginx.conf.
+  # You can create a ConfigMap with the key nginx.conf to override the default nginx.conf, and please
+  # make sure nginx server is listening on port 8080
+  configMapInfo:
```

Comment:
any specific configmap we needed to make it running for now?

Reply:
operator will create a confimap to store the nginx.conf, mount it to pod

Reply:
we can even generate.the conf on the fly in entrypoint with necessary env var injected by operator.

Reply:
> if we are not going to allow user to provide their own nginx conf, is this configmap needed in crd?

no, shall we delete it from values.yaml ?

Reply:
operator allow users to use their own nginx.conf, in case our implementation is not comlete.

Reply:
I don't think so, I can't think of any cases that the user needs to modify this nginx.conf. In order not to build our own nginx img, we can do the nginx conf mounted from configmap by the operator. the img is necessary in case they can't access the img from public network.

Reply:
ok, I will remove it.